pkgs/dart_mcp_server/lib/src/mixins/pub.dart

```diff
@@ -87,7 +87,15 @@ base mixin PubSupport on ToolsSupport, LoggingSupport, RootsTrackingSupport
     inputSchema: Schema.object(
       properties: {
         ParameterNames.command: Schema.string(
-          title: 'The pub command to run.',
+          title: '''The pub subcommand to run.
```

Comment:
See the `add` command below, it has a TODO for supporting dev dependencies etc, does this resolve that?